edgexfoundry/edgex-compose
# fix: [Levski] Update ASC version for recent patch release

**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-compose/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

  <link to docs PR>


## Testing Instructions
from this branch run ` make run no-secty app-sample` in top folder to run the pre-built compose file.
Verify ASC services running are 2.3.1 version
